modules/frontend/v1/frontend.go

```diff
@@ -22,41 +23,34 @@ import (
 	"github.com/grafana/tempo/modules/frontend/queue"
 	"github.com/grafana/tempo/modules/frontend/v1/frontendv1pb"
 	"github.com/grafana/tempo/pkg/util"
-	"github.com/grafana/tempo/pkg/validation"
 )
 
 // Config for a Frontend.
 type Config struct {
 	MaxOutstandingPerTenant int                    `yaml:"max_outstanding_per_tenant"`
-	QuerierForgetDelay      time.Duration          `yaml:"querier_forget_delay"`
 	MaxBatchSize            int                    `yaml:"max_batch_size"`
 	LogQueryRequestHeaders  flagext.StringSliceCSV `yaml:"log_query_request_headers"`
 }
 
 // RegisterFlags adds the flags required to config this to the given FlagSet.
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&cfg.MaxOutstandingPerTenant, "querier.max-outstanding-requests-per-tenant", 2000, "Maximum number of outstanding requests per tenant per frontend; requests beyond this error with HTTP 429.")
-	f.DurationVar(&cfg.QuerierForgetDelay, "query-frontend.querier-forget-delay", 0, "If a querier disconnects without sending notification about graceful shutdown, the query-frontend will keep the querier in the tenant's shard until the forget delay has passed. This feature is useful to reduce the blast radius when shuffle-sharding is enabled.")
```

Comment:
remove it from docs as well? and mark it as breaking change in case folks are using it?